CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Add the `update_brand()` method to the User object
 * Removes `_max_timeout` and instead uses a flat 60 second timeout for requests
 * Adds Python version to user-agent header on requests
+* Removes `shipment.get_rates()` method since the shipment struct already has rates. If you need to get new rates for a shipment, please use the `shipment.regenerate_rates()` method.
```

Comment:
remove the "struct"

Reply:
Resolved in 9b78a5e3de58c64bb493919aef40208a42db7e51